src/SVector.jl

```diff
@@ -117,3 +117,12 @@ macro SVector(ex)
         error("Use @SVector [a,b,c], @SVector Type[a,b,c] or a comprehension like [f(i) for i = i_min:i_max]")
     end
 end
+
+# Named field access for the first three elements with the conventional
+# field names
+function Base.getproperty(v::Union{SVector}, name::Symbol)
```

Comment:
Unnecessary Union

Reply:
Heh oops, thanks!

The reason this was there was that I had this defined as `Union{SVector,MVector}` because the implementations are the same.

But then I didn't know which file to put these in so I duplicated them instead as they're a concrete interface. Kind of a stupid reason to have duplicate code though I guess...